src/models/expenses.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const sequelize = require('../dataBase');
+
+const Expense = sequelize.define('expense', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.UUID,
+    primaryKey: true,
+  },
+  title: {
+    type: DataTypes.STRING,
+    primaryKey: false,
```

Comment:
false it isn't necessary, you can delete all primaryKey: false

Reply:
Done